webapp/src/main/java/org/opentestsystem/rdw/admin/web/GroupController.java

```diff
@@ -27,9 +31,14 @@
         return service.getByQuery(user, query);
     }
 
+    @PutMapping("/api/groups")
```

Comment:
Do we want to use delete semantics all the way up until the repo implementation and let update/soft delete be the impl or do we want the client to know that it is soft deleting?
I'm not sure what philosophy we have here.